include/eve/arch/riscv/rvv_utils.hpp

```diff
@@ -36,5 +36,10 @@ constexpr auto rvv_logical_ratio_v = []
 
 template<plain_scalar_value T>
 using rvv_m1_wide = wide<T, fixed<__riscv_v_fixed_vlen / 8 / sizeof(T)>>;
+
+// Represents a wide that occupies the same size, but formed with int8_t.
+// Here we use that sizeof(std::int8_t) == 1, so sizeof(In) will represent the number
+// of std::int8_t, that are needed to have the same size.
+template<arithmetic_simd_value In> using rvv_i8_wide = wide<std::int8_t, fixed<sizeof(In)>>;
```

Comment:
For chars, rvv is not special.
You can always cast to `wide<std::uint8_t, eve::fixed<sizeof(T) * N::value>`

I'm ok with having some typedef for `wide_bytes<T>` or smth, but it's not a rvv specific thing.
@jfalcou  - where would we put one.

Reply:
We don't a specific place for those. So I would say in wide.hpp after the main definition ?

Reply:
Done